### PR TITLE
doc: core/config documentation

### PR DESCRIPTION
**Pull request by mGGk-fr**
# What

This MR adds doc for `@front-commerce/core/config`

# Demo

https://deploy-preview-782--heuristic-almeida-1a1f35.netlify.app/docs/remixed/api-reference/front-commerce-core/config

### CODE REVIEW

File: docs/04-api-reference/front-commerce-core/config.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  return getCurrentShopConfig(config);
+}
+
+```
+
+## UserConfig
+
+### `extensions`
+
+_Optionnal_ Array of extensions to load
```
Comment (real34) [inline docs/04-api-reference/front-commerce-core/config.mdx:67]:
```suggestion
_Optional_ List of extensions to load
```

File: docs/04-api-reference/front-commerce-core/config.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+### `stores`
+
+**Required** [Store configuration](#)
```
Comment (real34) [inline docs/04-api-reference/front-commerce-core/config.mdx:71]: **issue:** dead link

**suggestion:** remove link until it's documented. In the meantime, you can add an example here (using an import).

If no issue exists: create one in Linear and add this as a reminder

File: docs/04-api-reference/front-commerce-core/config.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+### `cache`
+
+**Required** Your [cache configuration](#)
```
Comment (real34) [inline docs/04-api-reference/front-commerce-core/config.mdx:75]: issue: dead link

suggestion: remove link until it's documented. In the meantime, you can add an example here (using an import).

If no issue exists: create one in Linear and add this as a reminder

File: docs/04-api-reference/front-commerce-core/config.mdx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+### `configuration`
+
+Object that define app configuration
+
+#### `providers`
+
+List of configuration providers to automatically register
```
Comment (real34) [inline docs/04-api-reference/front-commerce-core/config.mdx:83]:
```suggestion
List of configuration providers to register in the application.
```

**Review by real34 — CHANGES_REQUESTED** [review]

File: docs/04-api-reference/front-commerce-core/config.mdx [hunk 5]
```diff
@@ -0,0 +1,107 @@
+---
+title: "config"
+sidebar_position: 6
+description: "Config related methods"
+---
+
+<p>{frontMatter.description}</p>
+
+## `defineUserConfig`
```
Comment (real34) [inline docs/04-api-reference/front-commerce-core/config.mdx:9]:
```suggestion
## `defineConfig`
```

**Review by real34 — APPROVED** [review]

File: docs/04-api-reference/front-commerce-core/config.mdx (hunk 1 above)
Comment (mGGk-fr) [reply docs/04-api-reference/front-commerce-core/config.mdx]: fix with your suggestion

File: docs/04-api-reference/front-commerce-core/config.mdx (hunk 4 above)
Comment (mGGk-fr) [reply docs/04-api-reference/front-commerce-core/config.mdx]: fixed with your suggestion

File: docs/04-api-reference/front-commerce-core/config.mdx (hunk 5 above)
Comment (mGGk-fr) [reply docs/04-api-reference/front-commerce-core/config.mdx]: fixed with your suggestion

File: docs/04-api-reference/front-commerce-core/config.mdx (hunk 3 above)
Comment (mGGk-fr) [reply docs/04-api-reference/front-commerce-core/config.mdx]: fixed with your suggestion and created FC-1751 for follow-up

File: docs/04-api-reference/front-commerce-core/config.mdx (hunk 2 above)
Comment (mGGk-fr) [reply docs/04-api-reference/front-commerce-core/config.mdx]: fixed with your suggestion and created FC-1752 for follow-up

Comment (real34) [reply docs/04-api-reference/front-commerce-core/config.mdx]: Thanks!

**Pull request merged**